src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/models/GameInstance.kt

```diff
@@ -0,0 +1,37 @@
+package com.gamingbacklog.api.gamingbacklogapi.models
+
+import lombok.Data
+import lombok.NoArgsConstructor
+import org.bson.types.ObjectId
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
+import org.springframework.data.mongodb.core.mapping.Field
+import java.util.*
+
+@Document("gameInstances")
+@NoArgsConstructor
+@Data
+data class GameInstance(
+  @Id
+  val id: String = ObjectId.get().toString(),
+
+  // igdb-specific data
+  @Field
+  val igdbId: String,
+  val name: String,
+  val platforms: List<String>,
+  val genres: List<String>,
+  val universes: List<String>,
+  val companies: List<String>,
+  val releaseDate: List<String>,
+  val images: List<String>,
+
+  // user-custom data
+  var rating: Int? = null,
+  var review: String? = null,
+  var ranking: String? = null,
+  var yearPlayed: Int? = null,
+  var yearReceived: Int? = null,
```

Comment:
Added! I think these are also very subject to change/we can add more down the line too

Reply:
I'm not testing this currently because it was annoying to do so, it's a temporary implementation for now but I'll clean it up once things get more official/as we go